src/qw/github.py

```diff
@@ -32,7 +32,7 @@ class Service(qw.service.GitService):
     def __init__(self, conf):
         """Log in with the gh auth token."""
         super().__init__(conf)
-        token = keyring.get_password("gh:github.com", "")
+        token = keyring.get_password("gh:github.com", "qw-pat")
```

Comment:
System "gh:github.com" with user "" is getting the password that `gh auth login` sets. If we aren't doing that, we should set the system perhaps to "qw" and the user to "" or the actual user's name. Otherwise we are claiming to have a gh password.
So we could continue to get "gh:github.com", "" if it exists if we want a simpler way to log in (for `gh` users) and then fall back to something else...
But probably the best thing is to make this line `token = keyring.get_password("pw", getpass.getuser())`
and do the same thing for the `login` command with `keyring.set_password`

Reply:
ah I think it must be set somewhere else differently in mac then. Okay easy, I'll update the get and when we create the login we can deal with the setting of the password

Reply:
though actually if a user had multiple QW projects then that might be an issue if they're using fine grained PATs as we suggested. Perhaps username should be `${organsiation}/{repo}` so that its unique to a specific project

Reply:
OK, this is a good idea as we are dealing with safety critical stuff and we don't want a worry that a qw dealing with one repo might access another.